eZ/Publish/Core/MVC/Legacy/Tests/Image/AliasCleanerTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * File containing the AliasCleanerTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\MVC\Legacy\Tests\Image;
+
+use eZ\Publish\Core\MVC\Legacy\Image\AliasCleaner;
+use PHPUnit_Framework_TestCase;
+
+class AliasCleanerTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var AliasCleaner
+     */
+    private $aliasCleaner;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
```

Comment:
Nitpick again: I'd typehint both types:

```
/**
 * @var eZ\Publish\Core\FieldType\Image\AliasCleanerInterface|\PHPUnit_Framework_MockObject_MockObject
 */
```

It pleases the completion spirits ;-)